src/page-multi-facility/ScenarioLibraryModal.tsx

```diff
@@ -1,19 +1,19 @@
 import React, { useEffect, useState } from "react";
 import styled from "styled-components";
 
-import { duplicateScenario, getScenarios } from "../database";
+import { deleteScenario, duplicateScenario, getScenarios } from "../database";
 import Colors from "../design-system/Colors";
 import { DateMMMMdyyyy } from "../design-system/DateFormats";
 import iconSrcCheck from "../design-system/icons/ic_check.svg";
 import iconSrcRecidiviz from "../design-system/icons/ic_recidiviz.svg";
+import { StyledButton } from "../design-system/InputButton";
 import Loading from "../design-system/Loading";
 import Modal, { Props as ModalProps } from "../design-system/Modal";
+import ModalDialog from "../design-system/ModalDialog";
 import PopUpMenu from "../design-system/PopUpMenu";
 import useScenario from "../scenario-context/useScenario";
 import { Scenario } from "./types";
 
-type Props = Pick<ModalProps, "trigger">;
```

Comment:
Just moved this down to be closer to the component declaration where it is used.